geoportal/c2cgeoportal_geoportal/lib/filter_capabilities.py

```diff
@@ -156,7 +156,9 @@ def caching_prepare_input_source(source, base=None):
             args = (source,) if base is None else (source, base)
             return old_prepare_input_source(*args)
 
-        document = urlopen(full_uri, proxies=proxies)
+        opener = urllib.request.FancyURLopener(proxies)
+        with opener.open(full_uri) as stream:
+            document = stream.read()
```

Comment:
For my information, why this change?

Reply:
The old one don't works with Python3, reported by PyLint